config/settings.yml

```diff
@@ -358,3 +358,12 @@ discourse_ai:
   ai_rag_images_enabled:
     default: false
     hidden: true
+
+  ai_summary_persona_id:
+    default: ""
+    client: true
```

Comment:
These don't need to be `client: true` (because the browser doesn't need to access them)